memegen/routes/overview.py

```diff
@@ -0,0 +1,20 @@
+from flask import Blueprint, current_app as app, url_for, render_template
+# http://stackoverflow.com/questions/16061514/pylint-pylint-unable-to-import-flask-ext-wtf
+from flask.ext.api.decorators import set_renderers  # pylint: disable=E0611,F0401
```

Comment:
Or you could just do `from flask_api.decorators import set_renderers`. The `flask.ext` imports are a sort of hack.